tests/manager/manager-var-dump-001.phpt

```diff
@@ -45,10 +45,10 @@ object(MongoDB\Driver\Manager)#%d (%d) {
       ["is_passive"]=>
       bool(false)
       ["tags"]=>
-      array(0) {
+      object(stdClass)#%d (0) {
```

Comment:
Also, I noticed that `php_phongo_server_to_zval()` uses fresh stack-allocated `php_phongo_bson_state` variables for each `bson_to_zval()` call, while `php_phongo_writeresult_get_debug_info()` re-uses them. I don't see any harm in re-using them, as it would only be modified if the `__pclass` field is picked up and ODM behavior is activated (and I wouldn't expect that for this BSON data).